.gitignore

```diff
@@ -3,3 +3,4 @@ build
 .DS_STORE
 .env
 **/*.mo
+.idea
```

Comment:
hope you don't mind. but it's been bugging me a bit to not have this (I use it a lot for multi line editing)